evidence/pool_test.go

```diff
@@ -82,7 +82,7 @@ func TestEvidencePoolBasic(t *testing.T) {
 	next := pool.EvidenceFront()
 	assert.Equal(t, ev, next.Value.(types.Evidence))
 
-	const evidenceBytes int64 = 372
+	const evidenceBytes int64 = 374
```

Comment:
Can you remind me why we changed this?

Reply:
It was to fix `TestEvidencePoolBasic`. The original logic set `maxBytes` right at the size required. This fork makes slight changes to the protobuf encoding (I think it was the keys), so the _actual_ size of the protobuf-encoded evidence is two bytes bigger.

Let me know if you want me to dig deeper on where exactly this difference stems from (I think it's from the changes in the keys, but I'm not 100% sure)

Reply:
Resolved